core/deployment/src/main/java/io/quarkus/deployment/dev/IsolatedRemoteDevModeMain.java

```diff
@@ -90,7 +90,7 @@ private synchronized JarResult generateApplication() {
             //ok, we have resolved all the deps
             try {
                 AugmentResult start = augmentAction.createProductionApplication();
-                if (!start.getJar().getType().equalsIgnoreCase(PackageConfig.MUTABLE_JAR)) {
+                if (!start.getJar().getType().equalsIgnoreCase(PackageConfig.Type.MUTABLE_JAR.toString())) {
```

Comment:
```suggestion
                if (start.getJar().getType() != PackageConfig.Type.MUTABLE_JAR) {
```

Reply:
I started down that road, but that would mean changing the property type in [JarResult](https://github.com/quarkusio/quarkus/blob/main/independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/app/JarResult.java), and then that spiders out into multiple other places, so I was trying to limit the scope of the changes.

If we really need to do that, I'll make the changes; I just wasn't sure how far to take it.

Reply:
I think it makes sense to change it if we're assuming that the type is meant to be only the values provided by this enum

Reply:
Ugh, forgot about the dependencies...  we'd either have to move `PackageConfig` to the Quarkus Bootstrap Core project, or keep the `String` property because it'll create a circular dependency to make it depend on Quarkus Core Deployment.

Reply:
Ah you're right, in this case `PackageConfig` needs to live in Bootstrap Core. Are you okay with these changes @aloubyansky ?

Reply:
...or, just move the enum?